pkg/nsx/services/vpc/vpc.go

```diff
@@ -568,8 +580,31 @@ func (s *VPCService) CreateOrUpdateVPC(obj *v1alpha1.NetworkInfo) (*model.Vpc, *
 		return existingVPC[0], &nc, nil
 	}
 
+	// build NSX LBS
+	var createdLBS *model.LBService
+	// TODO(gran) use switch to enable/disable NSXLB and AVI
+	if s.NSXConfig.NsxConfig.NSXLBEnabled() {
+		lbsSize := s.NSXConfig.NsxConfig.GetNSXLBSize()
+		// TODO(gran) Uncomment after #628 merged
+		//if nc.LbServiceSize != "" {
+		//	lbsSize = nc.LbServiceSize
+		//}
+		vpcPath := fmt.Sprintf(VPCKey, nc.Org, nc.NsxtProject, nc.Name)
+		var relaxScaleValidation *bool
+		if s.NSXConfig.NsxConfig.RelaxNSXLBScaleValication {
```

Comment:
the default value of relaxScaleValidation is nil, so if s.NSXConfig.NsxConfig.RelaxNSXLBScaleValication is false, relaxScaleValidation is nil.
just let 
relaxScaleValidation := s.NSXConfig.NsxConfig.RelaxNSXLBScaleValication 
will make it simple.
and let buildNSXLBS parameter relaxScaleValidation type is bool instead of *bool

Reply:
Not get your point. From NCP code. `relaxScaleValidation` should be nil if `NsxConfig.RelaxNSXLBScaleValication` is not true, thus we cannot use 
`relaxScaleValidation := s.NSXConfig.NsxConfig.RelaxNSXLBScaleValication`
in this code, since user may set `NsxConfig.RelaxNSXLBScaleValication` to false explicitly.

Reply:
for wcp, relaxScaleValidation should always be true, so why L594 need to check the s.NSXConfig.NsxConfig.RelaxNSXLBScaleValication ?

Reply:
I just followed NCP logic here.